fe/fe-core/src/main/java/com/starrocks/external/elasticsearch/MappingPhase.java

```diff
@@ -49,6 +49,8 @@ public MappingPhase(EsRestClient client) {
     public void preProcess(SearchContext context) {
         if (context.version() != null && context.version().onOrAfter(EsMajorVersion.V_7_X)) {
             includeTypeName = true;
+        } else if (context.version() != null && context.version().before(EsMajorVersion.V_7_X)) {
+            includeTypeName = false;
```

Comment:
includeTypeName  default value is false， why add this line？

Reply:
When the es is down or crashed, the SR loses the connection with the ES , and the version is set to 7.x(EsMajorVersion.LATEST) in VersionPhase#execute function by default, and then the field is set to True (MappingPhase # preProcess). After the es is restarted and restored, the version number is restored to 6. x, but the includeTypeName is still true, not false.

I have emailed seaven to describe our specific scenario.

Reply:
@RowenWood